Giti Mafakheri/week-01/Day03/Function-Array/js/exercises.js

```diff
@@ -0,0 +1,81 @@
+// # Array and Functions Bonus Material
+//
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. Use the if-then-else construct available in Javascript.
+//You'll have to remember your pre-work, or do some googling to figure this out.
+
+const maxOfTwoNumbers = function(num1,num2){
```

Comment:
Technically this is correct. However for readability we'd prefer you to write it as: 
`const maxOfTwoNumbers = function (num1,num2) {`